modules/local/plot_run_gantt/tests/nextflow.config

```diff
@@ -0,0 +1,2 @@
+process { withName: 'SEQERA_RUNS_DUMP' { container = 'public.ecr.aws/seqera-labs/wave/containers:tower-cli-0.9.0--2cb0f2e9d85d026b' } }
```

Comment:
Why do we need this file? Because I added a separate `container.config`? Maybe we need to put this in the process for now and we can move it later.

Reply:
For some reason the global container config wasn't pulled into the the tests. I tried using `projectDir` on the config line where you included it, but that didn't do the trick. So I put it in the 'figure out later' box.

Reply:
Ok. I'll have a play locally. Thanks!